pkgs/development/python-modules/lightgbm/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv
+, fetchPypi
+, cmake
+# Python deps
+, numpy
+, scipy
+, scikitlearn
+, buildPythonPackage
+}:
+
+buildPythonPackage rec {
+  name = "${pname}-${version}";
+  pname = "lightgbm";
+  version = "2.0.10";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1f1drk3s9sfhscp3dvsxhrnn2ajhfrn65rz4gwl09lasy2b8s6wh";
+  };
+
+  propagatedBuildInputs = [ numpy scipy scikitlearn ];
+  buildInputs = [
+    cmake
+  ];
+
```

Comment:
```
Ran 0 tests in 0.000s
```
No tests are found. Either the test runner cannot find tests and needs to be patched, or there simply are no tests in which case the tests need to be disabled. Do include a comment explaining why the tests are disabled.

Reply:
I disabled tests and added a comment. The tests don't seem to be included in the pypi distribution. We could build from GitHub if we want the tests?